utils/build-script

```diff
@@ -1148,8 +1151,13 @@ class BuildScriptInvocation(object):
             print("--- Running tests for %s ---" % product_name)
             product.test(host_target)
             print("--- Finished tests for %s ---" % product_name)
+        # Install the product if it should be installed specifically, or
+        # if it should be built and `install_all` is set to True.
```

Comment:
This is no longer true. CMark I think does install things and is now an early product. So I would say products like early-swift-driver which are before_build_script_impl products that are never installed.